cnd/migrations/initial_metadata/up.sql

```diff
@@ -0,0 +1,9 @@
+CREATE TABLE metadatas (
+       swap_id VARCHAR(255) NOT NULL PRIMARY KEY,
```

Comment:
Not sure if we want to do this as part of this PR but:

I think it is best practise to have a "stupid" database-only primary key if you have a domain-specific identifier already. (primary keys should be numbers I think)
The primary key would be used for foreign-key things and the domain-identifier only needs to have a `UNIQUE` attribute.